beacon/beacon.js

```diff
@@ -3,38 +3,63 @@
 const { Command } = require('commander');
 const program = new Command();
 const promptly = require('promptly');
-const applyContrib = require('./src/apply');
-const branchName = require('current-git-branch');
+const applyContribution = require('./src/apply');
 const chalk = require('chalk');
-let circuits = ['deposit', 'burn', 'tokenise', 'transfer', 'withdraw'];
+const axios = require('axios');
 
-program.description('CLI').version('0.8.0');
+const CIRCUITS = process.env.CIRCUITS.split(',');
```

Comment:
if we are calling methods on something that may not exists, maybe we can optinonally chain these?

```js
const CIRCUITS = process.env?.CIRCUITS?.split(',') ?? [];
```